ksqldb-engine/src/test/java/io/confluent/ksql/engine/KsqlEngineTest.java

```diff
@@ -1392,29 +1392,6 @@ public void shouldNotThrowWhenPreparingDuplicateTable() {
     // Then: no exception thrown
   }
 
-  @Test
```

Comment:
These shouldn't be needed as in order to pass validation it needs to not fail the if not exists part